arch/risc-v/src/mpfs/mpfs_start.c

```diff
@@ -116,6 +116,10 @@ void __mpfs_start(uint64_t mhartid)
   const uint32_t *src;
   uint32_t *dest;
 
+  /* Configure FPU */
+
+  riscv_fpuconfig();
```

Comment:
Just to prevent some future change from breaking this, can you please add check for
```

if (mhartid != 0)
{
  /* Configure FPU */

  riscv_fpuconfig();
}
```

Reply:
Got it

Reply:
LGTM!